packages/flutter_tools/pubspec.yaml

```diff
@@ -119,4 +118,4 @@ dartdoc:
   # Exclude this package from the hosted API docs.
   nodoc: true
 
-# PUBSPEC CHECKSUM: 588f
+# PUBSPEC CHECKSUM: e70c
```

Comment:
The pubspec was updated through `flutter update-packages --force-upgrade` after editing my `~/.gitconfig` to include the fix described here: https://github.com/flutter/flutter/issues/144033#issuecomment-1961771483